src/js/pages/catalog/PackagesTab.js

```diff
@@ -226,12 +235,12 @@ class PackagesTab extends mixin(StoreMixin) {
         }" `;
 
         return (
-          <div className="clearfix">
+          <Trans render="div" className="clearfix">
             {noResults}
```

Comment:
Should we moved the text in this variable on line 233 to inside the this competent instead? it would make translating easier. 
instead of `{noResults} (<0>view all</0>)` the key would be 
`No results were found for your search: "{searchString}" (<0>view all</0>)`